src/test/lib/operators/join_test_runner.cpp

```diff
@@ -58,11 +58,16 @@ struct InputTableConfiguration {
   InputTableType table_type{};
   EncodingType encoding_type{EncodingType::Unencoded};
 
-  // Only for JoinIndex. Chunk range of indexed join column segments.
+  // Only for JoinIndex. Chunk range of indexed join column segments. Initially, this should have been an optional, but
+  // valgrind reported access to uninitialized memory in release builds (and ONLY in them!). We think it is a false
+  // positive.
```

Comment:
I only investigated the memcheck error with an optional IndexScope in more detail. So I cannot tell without further investigations if this is a false positive.